src/utils/object.js

```diff
@@ -44,3 +46,46 @@ export function map(opt_initial) {
 export function hasOwn(obj, key) {
   return hasOwn_.call(obj, key);
 }
+
+/**
+ * @param {!Object} a the destination object
+ * @param {!Object} b
```

Comment:
Nit: Instead of `a` and `b`, use `target` and `source` which are self-documenting.